tool/sync_default_gems.rb

```diff
@@ -446,14 +446,17 @@ def ignore_file_pattern_for(gem)
 
     # Gem-specific patterns
     case gem
+    when "rubygems"
+      # We don't copy any vcr_cassettes to this repository. Because the directory does not
+      # exist, rename detection doesn't work. So it starts with the original path `bundler/`.
+      %r[\A(?:
+        bundler/spec/support/artifice/vcr_cassettes
+      )\z]mx
     when "yarp"
+      # Rename detection never works between ruby/ruby/doc and ruby/yarp/docs
```

Comment:
Is this comment relevant to what we're doing here?

Reply:
Just moved as-is.
The rename detection seems working to me.
Something changed?

Reply:
These gem-specific patterns seem no longer needed.

Reply:
> These gem-specific patterns seem no longer needed.

They were necessary. I added them back in https://github.com/ruby/ruby/commit/43ab2acf8242c580b1ef5d644ea94d48669d4869.